### ISSUE TITLE
Galaxy Error

### ISSUE THREAD

**Issue by matnguyen** [tseemann/snippy#207]
When I try to run the Galaxy version of Snippy, I get the following error on all generated datasets:

```
Traceback (most recent call last):
  File "/galaxy-central/lib/galaxy/jobs/runners/__init__.py", line 170, in prepare_job
    job_wrapper.prepare()
  File "/galaxy-central/lib/galaxy/jobs/__init__.py", line 971, in prepare
    self.command_line, self.extra_filenames, self.environment_variables = tool_evaluator.build()
  File "/galaxy-central/lib/galaxy/tools/evaluation.py", line 454, in build
    raise e
KeyError: 'element_identifier'
```

Do you know what the problem might be?  Thanks!

**Comment by tseemann** [comment]
Sorry I can't help you. I don't control the snippy galaxy wrapper.

Maybe @slugger70 can advise.

**Comment by Slugger70** [comment]
What version of galaxy are you running?

**Comment by matnguyen** [comment]
I'm running version 17.01

**Comment by Slugger70** [comment]
Hi Matt, I think element.identifier only became available in Galaxy version 17.09 and above.

**Comment by matnguyen** [comment]
I have an in-house tool that uses element.identifier on the same Galaxy instance that works fine.
